core/services/offchainreporting/delegate.go

```diff
@@ -224,7 +225,7 @@ func (d Delegate) ServicesForSpec(jobSpec job.Job) (services []job.Service, err
 			runResults,
 			d.pipelineRunner,
 			make(chan struct{}),
-			jobSpec.ID,
```

Comment:
I removed `jobSpec.ID` here since loggerWith already contains the job ID (line 127)